src/Stack/Types/PackageDump.hs

```diff
@@ -31,4 +31,6 @@ data InstalledCacheEntry = InstalledCacheEntry
 instance Store InstalledCacheEntry
 
 installedCacheVC :: VersionConfig InstalledCacheInner
-installedCacheVC = storeVersionConfig "installed-v1" "GGyaE6qY9FOqeWtozuadKqS7_QM="
+-- FIXME would be really nice to avoid changing this hash...
+--installedCacheVC = storeVersionConfig "installed-v1" "GGyaE6qY9FOqeWtozuadKqS7_QM="
```

Comment:
@mgsloan Do you have any insight into a way to avoid having to change this hash? Unless I'm mistaken, changing it will correct the precompiled package cache.